### PR TITLE
Dev Pipeline - sort by most recent activity

### PR DESCRIPTION
**Pull request by MariagraziaAlastra**
@russellholt 
screenshot-maria duckduckgo com 5001 2015-09-03 21-29-58

### CODE REVIEW

**Comment by russellholt** [comment]
I think we really want to do the sorting in javascript, so that it can be reordered by a few metrics.
date of last activity (which is equivalent to this one - idle time), date of creation, and producer are three of the obvious ones.

For the prototyping purposes of *this week*, we can go with this. This is an example of blurring the lines between the front end and the back end, which is something we do not want. We will need to revisit this next week, and move the sorting to javascript on the front end.

File: lib/DDGC/Web/Controller/InstantAnswer.pm [hunk 1]
```diff
@@ -138,7 +138,12 @@ sub dev_pipeline_json :Chained('dev_pipeline_base') :PathPart('json') :Args(0) {
 
     my @ias;
     my $key;
-    @ias = $rs->search({'dev_milestone' => { '=' => ['planning', 'development', 'testing', 'complete']}});
+    @ias = $rs->search(
```
Comment (russellholt) [inline lib/DDGC/Web/Controller/InstantAnswer.pm]: the future intentions for this code should be noted in a comment, why it's here, etc.

**Comment by russellholt** [comment]
I haven't seen generated CSS going into the same PR like this here. Usually that goes into master, does it not?

**Comment by russellholt** [comment]
@jagtalon can you test this please?

**Comment by MariagraziaAlastra** [comment]
@russellholt I've chosen this solution because that's the easiest and the fastest to implement, for our prototyping scope of this week.

Of course that should be done on the front-end instead.

The generated css should be ignored by git automatically, I don't know what the deal is with it, I'll just remove that file.

File: lib/DDGC/Web/Controller/InstantAnswer.pm (hunk 1 above)
Comment (MariagraziaAlastra) [inline lib/DDGC/Web/Controller/InstantAnswer.pm]: Alright

**Comment by russellholt** [comment]
:+1:

**Pull request merged**